hail/python/hail/docs/change_log.md

```diff
@@ -1,5 +1,67 @@
 # Change Log
 
+## 0.2.13
+
+Released 2019-04-18
+
+Hail is now using Spark 2.4.x by default. If you build hail from source, you
+will need to acquire this version of Spark and update your build invocations
+accordingly.
+
+### New features
+
+- (hail#5828) Remove dependency on htsjdk for VCF INFO parsing, enabling
+  faster import of some VCFs.
+- (hail#5860) Improve performance of some column annotation pipelines.
+- (hail#5858) Add `unify` option to `Table.union` which adds special behavior
+  when duplicate rows are present.
+- (hail#5756) Hail now uses Spark 2.4.x by default.
+- (hail#5677) `MatrixTable` now also supports `show`.
+- (hail#5790) Add `array.head()` which returns the first element of the array,
+  or missing if the array is empty.
+- (hail#5690) Add `_localize` flag to `locus_windows` to prevent serialization
```

Comment:
I haven't been documenting underscored features. May we should promote the `localize` stuff to public?

Reply:
probs, I changed it to a bland statement about improved ld_matrix performance